### ISSUE TITLE
[BUG] After updating universal x86 tuning utility the application opens for a split second and immidiatly closes

### ISSUE THREAD

**Issue by M8M24** [JamesCJ60/Universal-x86-Tuning-Utility#231]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behaviour:

**Expected behaviour**

**Screenshots**

**Desktop (please complete the following information):**
 - OS Version: [e.g. Windows 11 22H2]

**Additional context**

**Comment by M8M24** [comment]
I have tried uninstalling and downloading it several times but the issue keeps reoccurring, i have tried different versions as well with the same result plus anything before version 2.3.0 Beta 1, says that their is a newer version available on my laptop, so i tried deleting any trace of it i could find and trying again but that didn't work, i also tried the remove universal x86 tuning utility option given by Universal.x86.Tuning.Utility.V2 i also tried the repair option but that didn't work either

**Comment by muthu-25** [comment]
I am facing the same issue on unmodified Windows 11 Home. My laptop has Ryzen 5 3500U

**Comment by rabiddoughnuts** [comment]
I am having the same issues on windows 11 home ryzen 7940hs. Until this latest update it worked and I was using it to undervolt

**Comment by hamixd** [comment]
same here. 3500u. latest windows. 2.0.0 runs fine though

**Comment by hamixd** [comment]
Ok guys I found the solution to this problem.

You all have to update to the latest chipset and graphics drivers for the software to work.

**Comment by rabiddoughnuts** [comment]
> Ok guys I found the solution to this problem.
> 
> You all have to update to the latest chipset and graphics drivers for the software to work.

That was it, I have a framework, so the new drivers weren't available till the 2nd, but it works now, thanks. Shoulda known, it's always drivers.

**Comment by muthu-25** [comment]
> Ok guys I found the solution to this problem.
> 
> You all have to update to the latest chipset and graphics drivers for the software to work.

Yup that did the trick, whilst my device was up to date, I needed to install drivers directly from AMD which I presume is still newer as manufacturer did not provide that.

**Comment by JamesCJ60** [comment]
Hi, please ensure that you are using the latest drivers from your hardware vendors and that all the software prerequisites have been installed.
